### ISSUE TITLE
Twitch

### ISSUE THREAD

**Issue by LegendNed** [lavalink-devs/Lavalink#322]
## Description
Attempting to load a twitch stream responds with an error.

## Version info
**Client used:** 1.3.50

Output of `java -version`:
```
openjdk version "13" 2019-09-17
OpenJDK Runtime Environment (build 13+33-Ubuntu-1)
OpenJDK 64-Bit Server VM (build 13+33-Ubuntu-1, mixed mode, sharing)

```

## Error:
```
5|Lavalink  | 2020-07-03 15:34 +02:00: 2020-07-03 15:34:38.681  INFO 18337 --- [  XNIO-1 task-5] l.server.player.AudioLoaderRestHandler   : GET /loadtracks
5|Lavalink  | 2020-07-03 15:34 +02:00: 2020-07-03 15:34:38.900  WARN 18337 --- [ader-2-thread-1] c.s.d.l.p.DefaultAudioPlayerManager      : Suspicious exception for loading item https://www.twitch.tv/20secondsviral
5|Lavalink  | 2020-07-03 15:34 +02:00:
5|Lavalink  | 2020-07-03 15:34 +02:00: com.sedmelluq.discord.lavaplayer.tools.FriendlyException: Server responded with an error.
5|Lavalink  | 2020-07-03 15:34 +02:00:  at com.sedmelluq.discord.lavaplayer.tools.io.HttpClientTools.fetchResponseAsJson(HttpClientTools.java:222) ~[lavaplayer-1.3.50.jar!/:na]
5|Lavalink  | 2020-07-03 15:34 +02:00:  at com.sedmelluq.discord.lavaplayer.source.twitch.TwitchStreamAudioSourceManager.fetchStreamChannelInfo(TwitchStreamAudioSourceManager.java:185) ~[lavaplayer-1.3.50.jar!/:na]
5|Lavalink  | 2020-07-03 15:34 +02:00:  at com.sedmelluq.discord.lavaplayer.source.twitch.TwitchStreamAudioSourceManager.loadItem(TwitchStreamAudioSourceManager.java:76) ~[lavaplayer-1.3.50.jar!/:na]
5|Lavalink  | 2020-07-03 15:34 +02:00:  at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.checkSourcesForItemOnce(DefaultAudioPlayerManager.java:437) ~[lavaplayer-1.3.50.jar!/:na]
5|Lavalink  | 2020-07-03 15:34 +02:00:  at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.checkSourcesForItem(DefaultAudioPlayerManager.java:419) ~[lavaplayer-1.3.50.jar!/:na]
5|Lavalink  | 2020-07-03 15:34 +02:00:  at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.lambda$createItemLoader$0(DefaultAudioPlayerManager.java:218) ~[lavaplayer-1.3.50.jar!/:na]
5|Lavalink  | 2020-07-03 15:34 +02:00:  at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[na:na]
5|Lavalink  | 2020-07-03 15:34 +02:00:  at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[na:na]
5|Lavalink  | 2020-07-03 15:34 +02:00:  at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[na:na]
5|Lavalink  | 2020-07-03 15:34 +02:00:  at java.base/java.lang.Thread.run(Thread.java:830) ~[na:na]
5|Lavalink  | 2020-07-03 15:34 +02:00: Caused by: java.lang.IllegalStateException: Response code from channel info is 410
5|Lavalink  | 2020-07-03 15:34 +02:00:  ... 10 common frames omitted
```

**Comment by schnapster** [comment]
Looks like a duplicate of #298 

According to it waiting on an LP fix here: https://github.com/sedmelluq/lavaplayer/pull/488
